resources/2to3migration.txt

```diff
@@ -0,0 +1,18 @@
+# version=1
+# If you update this file make sure you bump the above version number it must be higher then the last run.
+#oldkey;newkey;QGIS3Group
+browser/favourites;*;None
+MSSQL/connections/selected;*;None
+MSSQL/connections/DMSLaptop/service;*;None
```

Comment:
Leftover testing stuff?

Reply:
Haha oh yeah *headdesk*. Don't even need this, I should be able to just copy the full key over hopefully.